### PR TITLE
treehouses services url with networkmode info (fixes #649)

### LINKED ISSUE #649: services: wrong ip address url
```
root@sriharivishnu:~# treehouses services planet url both
169.254.4.234:80
169.254.4.234:2200 
...
```

### PR DESCRIPTION
**Pull request by kjong**
Fixes #649

Changed from using `hostname` to using `treehouses networkmode` to get IP address when creating local url for services.

### CODE REVIEW

File: modules/services.sh [hunk 1]
```diff
@@ -326,7 +326,7 @@ function services {
           if [ "$command_option" = "local" ]; then
             for i in $(seq 1 "$(get_port $service_name | wc -l)")
             do
-              local_url=$(hostname -I | head -n1 | cut -d " " -f1)
+              local_url=$(treehouses networkmode info | grep -oP -m1 '(?<=ip: ).*?(?=,)')
```
Comment (LordJashin32) [inline modules/services.sh]: Can't you just do this with ```networkmode "info"``` ? Cli.sh has these files included. On a manual or separate installation.  I think might not have access to ```treehouses``` in command line. 
https://github.com/treehouses/cli/blob/master/cli.sh#L208-L209

Comment (kjong) [reply modules/services.sh]: Yeah, I always forget NOT to use `treehouses`.

**Comment by LordJashin32** [comment]
Seems to work but It doesn't seem to give me a URL with TOR enabled. It just throws error again and again about how i have to do ```tor start``` and so i did ```sudo treehouses tor start``` and ```sudo cli.sh tor start``` and it was still throwing that error with planet running.

**Comment by kjong** [comment]
> Seems to work but It doesn't seem to give me a URL with TOR enabled. It just throws error again and again about how i have to do `tor start` and so i did `sudo treehouses tor start` and `sudo cli.sh tor start` and it was still throwing that error with planet running.

Do you have any tor ports added? If you try to use tor without any ports added, it will give errors, even after you do `tor start`.

**Comment by LordJashin32** [comment]
> > Seems to work but It doesn't seem to give me a URL with TOR enabled. It just throws error again and again about how i have to do `tor start` and so i did `sudo treehouses tor start` and `sudo cli.sh tor start` and it was still throwing that error with planet running.
> 
> Do you have any tor ports added? If you try to use tor without any ports added, it will give errors, even after you do `tor start`.

I think i had port 80 added when I ran it the last time because i suspected that too. Did i screw it up? Should i add ports before or after? Did i add the right port?

**Comment by kjong** [comment]
So it seems like tor is kind of finicky with how it wants to start up.

The most straightforward way to get tor started up after running `tor destroy` is to add a port, then start tor.

```
root@kjong:~/cli# treehouses tor destroy
Success: the tor service has been destroyed
root@kjong:~/cli# treehouses tor add 80
Success: the port has been added
root@kjong:~/cli# treehouses tor start
Success: the tor service has been started
root@kjong:~/cli# treehouses tor list
external <=> local
80  <=> 80
```

**Pull request merged**